src/clients/BinariesVersionClient.ts

```diff
@@ -74,12 +74,6 @@ export class BinariesVersionClient {
             this._logger.error(TelemetryEvent.BinariesVersionClient_GetDownloadInfoError, error);
             throw error;
         }
-        finally {
-            this._logger.trace(TelemetryEvent.BinariesVersionClient_GetDownloadInfoStatus, /*properties*/ {
-                binariesVersionsDownloadTimeInMilliseconds: new Date().getTime() - downloadStartTime.getTime(),
```

Comment:
downloadStartTime variable needs to be removed because it will be unsed ? @Vidya2606  can you please check ?

Reply:
ah I missed it! removed it now.